squid-blockexplorer/src/blocks/utils.ts

```diff
@@ -94,7 +94,7 @@ export function createCall(
 export function calcSpacePledged(solutionRange: bigint): bigint {
   const MAX_U64 = 2n ** 64n - 1n;
   const SLOT_PROBABILITY = [1n, 6n];
-  const PIECE_SIZE = 4096n;
+  const PIECE_SIZE = 32768n;
```

Comment:
This is not the same number as in https://github.com/subspace/subspace/blob/7093bc9d9fbfde52e211133ab2f59cf0519a872c/crates/subspace-core-primitives/src/lib.rs#L58